source/Mission.cpp

```diff
@@ -416,6 +430,14 @@ bool Mission::IsMinor() const
 
 
 
+void Mission::EnsureUUID()
+{
+	if(uuid.size() < 1)
```

Comment:
Most other sections (for ships and NPCs) use `if(uuid.empty())`. Is there a reason the check here is different?